app/forms/state_file/md_tax_refund_form.rb

```diff
@@ -6,16 +6,29 @@ class MdTaxRefundForm < QuestionsForm
                        :routing_number,
                        :account_number,
                        :account_type,
-                       :bank_name,
-                       :account_holder_name
+                       :account_holder_first_name,
+                       :account_holder_middle_initial,
+                       :account_holder_last_name,
+                       :account_holder_suffix,
+                       :has_joint_account_holder,
+                       :joint_account_holder_first_name,
+                       :joint_account_holder_middle_initial,
+                       :joint_account_holder_last_name,
+                       :joint_account_holder_suffix,
+                       :bank_authorization_confirmed
     set_attributes_for :confirmation, :routing_number_confirmation, :account_number_confirmation
 
     validates :payment_or_deposit_type, presence: true
 
-    with_options unless: -> { payment_or_deposit_type == "mail" } do
-      validates :account_holder_name, presence: true
+    SUFFIX_OPTIONS = %w[JR SR I II III IV V VI VII VIII IX X]
 
-      validates :bank_name, presence: true
+    with_options unless: -> { payment_or_deposit_type == "mail" } do
+      validates :account_holder_first_name, presence: true
+      validates :account_holder_last_name, presence: true
+      validates :account_holder_first_name, format: { with: /\A[a-zA-Z]{1}([A-Za-z\-\s']{0,15})\z/.freeze, message: ->(_object, _data) { I18n.t('errors.attributes.first_name.invalid_format') }}
+      validates :account_holder_last_name, format: { with: /\A[a-zA-Z]{1}([A-Za-z\-\s']{0,31})\z/.freeze, message: ->(_object, _data) { I18n.t('errors.attributes.last_name.invalid_format') }}
```

Comment:
ask Tiffany about following schema strictly? (allowing numbers in first/last name)

Reply:
https://codeforamerica.atlassian.net/browse/FYST-526?focusedCommentId=13466

Reply:
answered! 
https://codeforamerica.atlassian.net/browse/FYST-526?focusedCommentId=13486
>@Arin Choi great catch--let’s allow alphabet, apostrophes and dash, and white space only. no numbers